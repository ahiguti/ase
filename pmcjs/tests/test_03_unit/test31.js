
/* checks ht cleanness */

var ht = Perl.NewHash();

var names = [
  '__proto__',
  '__parent__',
  '__count__',
  'proto',
  '__noSuchMethod__',
];

for (var i = 0; i < names.length; ++i) {
  var n = names[i];
  print(ht[n]);
}

for (var i = 0; i < names.length; ++i) {
  var n = names[i];
  ht[n] = 'VAL: ' + n;
}

for (var i = 0; i < names.length; ++i) {
  var n = names[i];
  print(ht[n]);
}
