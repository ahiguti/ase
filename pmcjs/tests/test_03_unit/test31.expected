undefined
undefined
undefined
undefined
undefined
VAL: __proto__
VAL: __parent__
VAL: __count__
VAL: proto
VAL: __noSuchMethod__
